src/services/tx/tx-sender/sdk.ts

```diff
@@ -98,12 +98,16 @@ export const getAssertedChainSigner = async (
  * dealing with smart-contract wallet owners
  */
 export const getUncheckedSafeSDK = async (onboard: OnboardAPI, chainId: SafeInfo['chainId']): Promise<Safe> => {
+  // console.log('getUncheckedSafeSDK', onboard, chainId)
   const signer = await getAssertedChainSigner(onboard, chainId)
   const sdk = getAndValidateSafeSDK()
 
   const ethAdapter = new EthersAdapter({
     ethers,
-    signerOrProvider: signer.connectUnchecked(),
+    // TODO: in etherhs v6 this method is no longer existing
+    // we should find out if this is still necessary
+    // signerOrProvider: signer.getUncheckedSigner(),
+    signerOrProvider: signer,
```

Comment:
There is no unchecked singer anymore. 
How can we figure out if we still need this? Where can I test this?
@usame-algan @schmanu

Reply:
This can be tested when having a safe be owner of another safe and trying to sign a transaction. With an unchecked signer it would immediately resolve the promise and close the tx-flow even if the transaction of the signer safe hasn't been executed yet.

Reply:
My best guess without testing is that it will resolve with the `safeTxHash` instead of a `txHash`.

If that is the case we should probably just put the tx into pending state without any timeout.